### PR TITLE
CSO Project

### PR DESCRIPTION
**Pull request by gvaldez8**
Transferring CSO project wireframes to code.

### CODE REVIEW

**Comment by gvaldez8** [comment]
Hi @epenzeymoog, I managed to code three pages thus far. Could you please take a look? I spent some time refactoring the code yesterday to be more modular but could use some feedback.

In the meantime, I will work on making these pages responsive and then circle back to creating the navigation's drop downs. Thank you!

File: source/index.html.erb [hunk 1]
```diff
@@ -1,92 +1,85 @@
...
+  <img class="header__image" src='./images/events-banner.png' />
+  <section class="header__event">
+      <h1>UPCOMING EVENTS</h1>
+
+      <div class="header__event--date">
+        <h1>03</h1>
+        <h3>JUN</h3>
+      </div>
+
+      <div class="header__event--details">
```
Comment (epenzeymoog) [inline source/index.html.erb]: it looks like some of these `<div>`s can be `<section>`s instead. It's ok to nest `<section>`s.

This resource is a little long and intense, but worth looking at when you have a moment: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines

File: source/index.html.erb [hunk 2]
```diff
@@ -1,92 +1,85 @@
...
-      <hr />
-     <% page_articles.each_with_index do |article, i| %>
-      <li>
-        <%= link_to article.title, article %>
-      </li>
-      <% end %>
-    </ul>
-  </section>
+      <div class="link__highlight--bottom">
+        <div class="helper__space--top"></div>
```
Comment (epenzeymoog) [inline source/index.html.erb]: These `helper__space` classes should be used on a `<div>` (or other element) that already exists and has content in it, as opposed to an empty `<div>` that only has this class. I would expect to see this class on the `<h4>` that occurs right after this `<div>` (if that doesn't make sense we can do a quick slack call).

File: source/index.html.erb [hunk 3]
```diff
@@ -1,92 +1,85 @@
...
 
-  <section class='books__container' id="books">
-    <h2>Read Me</h2>
-    <ul>
-      <h3>Currently Reading</h3>
-      <hr />
-      <li><em>Git for Humans</em> by David Demaree</li>
-      <li><em>Accessibility for Everyone</em> by Lara Kalbag</li>
-    </ul>
+  <img class="image__right" src='./images/opportunities.png' />
```
Comment (epenzeymoog) [inline source/index.html.erb]: don't forget your `alt` attribute!

File: source/index.html.erb [hunk 4]
```diff
@@ -1,92 +1,85 @@
...
+  <section class="form__subscribe">
+    <h1>SUBSCRIBE</h1>
+    <h2>Stay updated through our email list on all events and news related to the Chicago Symphony Orchestra!</h2>
+
+    <form>
+      <fieldset>
+        <label class="active" for="name">Name</label>
+        <input id="Name" type="text" class="name" placeholder="John Doe">
+        
+          <div class="helper__space--top"></div>
```
Comment (epenzeymoog) [inline source/index.html.erb]: go through and put all these `helper__space` classes onto an existing element.

File: source/layouts/_footer.erb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    <li><a href="#">Civic Orchestra</a></li>
+  </ul>
+
+  <ul>
+    <li><a href="#">Music Advocacy</a></li>
+    <li><a href="#">CSO Auditions</a></li>
+    <li><a href="#">CSO Administration Association</a></li>
+    <li><a href="#">Facility Rentals</a></li>
+  </ul>
+
```
Comment (epenzeymoog) [inline source/layouts/_footer.erb]: this is nitpicky, but get rid of this extra line :)

File: source/layouts/layout.erb [hunk 6]
```diff
@@ -1,22 +1,21 @@
...
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Use the title from a page's frontmatter if it has one -->
+    <title><%= current_page.data.title || "Middleman" %></title>
+    <%= stylesheet_link_tag "site" %>
+    <%= javascript_include_tag "site" %>
   </head>
-
+  
+  <%= partial "layouts/navigation" =%>
```
Comment (epenzeymoog) [inline source/layouts/layout.erb]: great use of partials!

**Comment by gvaldez8** [comment]
Hey @epenzeymoog, trying to make another pull request to get your feedback or suggestions like we mentioned during our IPM, but can I do that when there's already one open?

**Pull request merged**